kvmagent/kvmagent/test/shareblock_testsuite/test_sharedblock_metadata_auto_repair.py

```
@@ -0,0 +1,171 @@
+
+    def gllk_repair_test(self):
+        offset = 65*1024**2
+        self.corrupt_data(vgUuid, offset, length=1048576)
+        lvm.fix_global_lock()
+        r, o = bash.bash_ro("vgs 2>&1")
+        self.assertEqual(False, "global lock" in o, str(o))
+
+        bash.bash_r("lvmlockctl -D %s" % vgUuid)
+        self.corrupt_data(vgUuid, offset, length=1048576)
+        lvm.fix_global_lock()
+        r, o = bash.bash_ro("vgs 2>&1")
```

Ruff (0.15.2)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)